src/Interfaces/GatewayInterface.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace CraigPaul\Moneris\Interfaces;
+
+interface GatewayInterface
+{
+    /**
+     * Capture a pre-authorized a transaction.
+     * @param \CraigPaul\Moneris\Transaction|string $transaction
+     * @param string|null $order
+     * @param mixed|null $amount
+     * @return \CraigPaul\Moneris\Response
+     */
+    public function capture ($transaction, $order = null, $amount = null);
+
+    /**
+     * Create a new Vault instance.
+     * @return \CraigPaul\Moneris\Vault
+     */
+    public function cards ();
```

Comment:
Couldn't this just be called `vault ()` instead?

Reply:
I don't want to actually change the API at all for BC considerations.